circle.yml

```diff
@@ -43,23 +43,21 @@ jobs:
       - run:
           name: Install Dependencies
           command: |
+            sed -i 's/libmysqlclient/default-libmysqlclient/' debian_packages.lst
+            make install_system_dependencies
+            sudo ln -s /usr/lib/x86_64-linux-gnu/libmysqlclient.so /usr/lib/x86_64-linux-gnu/libmysqlclient.so.18
+            bin/install-supported-firefox
```

Comment:
It would be nice if the logic for installing firefox on local devstack was also moved to install-supported-firefox sometime. It's currently fragmented between this script and the makefile.

Reply:
I couldn't agree more @swalladge 

I moved all debian/ubuntu packages installation to the Makefile with that in mind, let's see if I end up having time this week to move Firefox installation as well.

Reply:
Maybe this can be done later in another ticket in the interest of unbreaking master sooner?